base/sysinfo.jl

```diff
@@ -385,10 +376,7 @@ See documentation in [Handling Operating System Variation](@ref).
 
 !!! note
     Not to be confused with `Sys.isbsd()`, which is `true` on DragonFly but also on
-    other BSD-based systems. `Sys.isdragonfly()` refers only to DragonFly.
-!!! compat "Julia 1.1"
-    This function requires at least Julia 1.1.
-"""
+    other BSD-based systems. `Sys.isdragonfly()` refers only to DragonFly."""
```

Comment:
To minimise the diff
```suggestion
    other BSD-based systems. `Sys.isdragonfly()` refers only to DragonFly.
"""
```